src/FakeItEasy/Expressions/ArgumentConstraints/EqualityArgumentConstraint.cs

```diff
@@ -33,12 +31,11 @@ public override string ToString()
                 writer.WriteArgumentValue(this.expectedValue);
                 return writer.Builder.ToString();
             }
-            catch (Exception ex) when (!(ex is UserCallbackException))
+            catch (Exception ex) when (ex is not UserCallbackException)

```

Comment:
Technically a bonus change, but I was right here. Will revert if you like.

Reply:
No, it's fine!